tests/test_data/peak_faction_workingday_response.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Rename this to `peak_faction_no_holiday_response`

Reply:
done